### ISSUE TITLE
Questions about HyperDenseNet configuration

### ISSUE THREAD

**Issue by rabin9948** [josedolz/LiviaNET#15]
Hi.

Continuously studying your HyperDenseNet, I have a few questions. 

1) Are T1 and Flair concatenated before the input of conv_1 in HyperDenseNet?

2) Also are T1 and Flair used for the input of all subsequent layers?

3) Is order important when concatenating outputs of all preceding layers from both paths?

4) The initial learning rate to 0.001 is reduced by a factor of 2 after every 5 epochs ( starting from epoch 10 ). So, when epoch is 10, the learning rate is 0.001*0.5. Next when epoch is 15, the learning rate is 0.001 * 0.5 * 0.5. Is that way right? also Is that epoch subepoch?

Thank you!

**Comment by josedolz** [comment]
Hi @rabin9948 

First of all, this is not the code of HyperDenseNet.

For your questions:

1) and 2) In HyperDenseNet, each image modality is treated separately in independent paths. Then, feature maps from each layer are concatenated to be fed as input of subsequent layers, in all the paths. Only feature maps are employed as input of next layers. 

3) This is something I was thinking when I implemented the dense connectivity pattern in HyperDenseNet. Even though I think it shouldn't change the performance, it might be interesting to evaluate it.

4) Yes, you are right for the learning rate decay. And epoch is epoch, not sub-epoch. So, when the training has  done 10 epochs, the LR will be reduced to half of its current value.

**Comment by rabin9948** [comment]
Thank you for answers! 
And I am sorry to leave the questions here.
